pkg/apply/task/delete_inv_task_test.go

```diff
@@ -29,4 +31,21 @@ func TestDeleteInvTask(t *testing.T) {
 	if result.Err != nil {
 		t.Errorf("unexpected error running DeleteInvTask: %s", result.Err)
 	}
+	// Hard-code error from fake inventory client, and make sure
```

Comment:
Can we set this up as table-driven tests?

Reply:
Done